packages/synthetix-main/contracts/interfaces/IMarketManagerModule.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 /// @title Market Manager Module
 interface IMarketManagerModule {
-    event MarketRegistered(address indexed market, uint128 indexed marketId);
+    event MarketRegistered(address indexed market, uint128 indexed marketId, address indexed sender);
```

Comment:
feels like sender might make more sense as the first property here

Reply:
I think we should keep this consistent with the other events.